keyboards/pinky/3/3.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "../pinky.h"
```

Comment:
You don't actually need the ".." here. 

When compiling, both the subfolder and main folder (eg, pinky, and 3 in this case) are added to the "common path".  So, you can call "pinky.h" directly, without having to use it's relative path. 

It's fin to do this, but "for the future"... it's something to keep in mind.

Reply:
I understood. I'll keep that in mind.